include/rcutils/logging.h

```diff
@@ -571,6 +571,26 @@ void rcutils_logging_console_output_handler(
     } \
   } while (0)
 
+/**
+ * \def RCUTILS_LOGGING_AUTOINIT_WITH_ALLOCATOR
+ * \brief Initialize the rcl logging library with allocator.
+ * Usually it is unnecessary to call the macro directly.
+ * All logging macros ensure that this has been called once.
+ */
+#define RCUTILS_LOGGING_AUTOINIT_WITH_ALLOCATOR(alloc)       \
```

Comment:
This is the same counterpart between `rcutils_logging_initialize` and `rcutils_logging_initialize_with_allocator`.

Reply:
@clalancette that is good idea, thanks!

Reply:
addressed with https://github.com/ros2/rcutils/commit/1385a23032c230525ec1cd65e57508adf1951e19